injection/session-injection.js

```diff
@@ -145,9 +145,7 @@ class SessionInjection {
         deepCloneMerge.extend(req.session, rawSessionValues);
     }
 
-    middlewareHandler(req, res, next) {
-        debug('middlewareHandler');
-
+    middlewareDecodePayload(req, res, next) {
         let payload;
```

Comment:
Add a `debug('middlewareDecodePayload')` line for consistency

Reply:
I'll add this in